engine/apps/schedules/models/on_call_schedule.py

```diff
@@ -500,7 +500,7 @@ def refresh_ical_final_schedule(self):
                     # check if event was ended or cancelled, update ical
                     dtend = component.get(ICAL_DATETIME_END)
                     dtend_datetime = dtend.dt if dtend else None
-                    if dtend_datetime and type(dtend_datetime) == datetime.date:
+                    if dtend_datetime and type(dtend_datetime) is datetime.date:
```

Comment:
the bump in `black` from `23.7.0` to `23.11.0` was complaining that these type assertions should instead use `is` / `is not`